kubo/010-set-conf.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/env sh
+
+set -eu
+
+ipfs config --json Import.CidVersion 0
+ipfs config --json Import.UnixFSRawLeaves false
+ipfs config Import.UnixFSChunker size-262144
+ipfs config Import.HashFunction sha2-256
+
+ipfs config Addresses.Gateway /ip4/0.0.0.0/tcp/$GATEWAY_PORT
```

Comment:
lets provide this envs to Oracle

Reply:
Provided via the corresponding `KUBO_` variables.